src/platform/datapath_raw_xdp_win.c

```diff
@@ -1734,6 +1911,22 @@ CxPlatXdpTx(
         Queue->Error = TRUE;
     }
 
+    if (XskRingOffloadChanged(&Queue->TxRing) &&
```

Comment:
What happens if offload change races with us submitting packets without xsum computed to xdp? Does xdp drop or compute on behalf of the caller?

Reply:
XDP never compares the current hardware state, it just sets the OOBs and lets the packets rip.

Reply:
Ok assuming that wouldn't cause the NIC to freak out

Reply:
As always in the Windows ecosystem, there are no guarantees, and we do know some NICs reuse/abuse OOBs they don't indicate support for. To a limited extent, the point of XDP is to avoid the runtime and maintenance costs of supporting rare/non-compliant drivers, so we're assuming NICs used in XDP scenarios will not freak until proven otherwise.